### ISSUE TITLE
Will this project continue?

### ISSUE THREAD

**Issue by jiangwei007** [thebracket/bgame#139]
Its a amazing game project , so will this project continue?

**Comment by thebracket** [comment]
Yes, but not soon. I had to pause when I broke a whole bunch of bones and the combination of pain medication and not being able to use my right hand left me unable to continue. Unfortunately, while on pain medication I went into "experiment" mode and tested out using UE4 to render things. That was a BAD idea, and won't continue - but I'm itching to get back to Nox Futura as soon as things calm down a bit with my Rust projects.

I've actually been considering porting some of this over to Rust, just because it's where I'm happier nowadays. It would be a lot of fun to combine that with another tutorial series, showing how to do it.

TLDR: It's not dead, but resurrection may take a while.

**Comment by jiangwei007** [comment]
I am so sorry for hear that , i want to send you my best wishes for your quick return to good health.

**Comment by jiangwei007** [comment]
And i'm interesting about you said:" using UE4 to rendering is a BAD idea", why?
